tests/e2e/initialization/node/main.go

```diff
@@ -0,0 +1,7 @@
+package main
+
+// Single-node initialization is not implemented yet This stub is created
+// as an intermediary step before final result.
+func main() {
+	panic("not implemented")
```

Comment:
This is a stub for now. The logic is to be added in a separate PR.

This is how we are going to spin up nodes for testing state-sync

Reply:
Err, why a main package at all?

Want to go ahead and add this in whichever PR you add main package logic?

Reply:
Chain initialization and node initialization are run as scripts via Docker. `main` package is the only entry point to a Go program. That's why we need it

chain init is a more complete example of how this is going to look in the end:
https://github.com/osmosis-labs/osmosis/blob/main/tests/e2e/chain_init/main.go

And yes, I can remove and readd this in another PR

Reply:
Done